Description: generate_init didn't found "shared" module
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-03-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tools/tr1/generate_init
+++ b/tools/tr1/generate_init
@@ -1,7 +1,12 @@
 #!/usr/bin/env python3
 import argparse
+import sys
+import os
 from pathlib import Path
 
+# Ajoute le répertoire parent au PYTHONPATH
+sys.path.insert(0, str(Path(__file__).resolve().parent.parent))
+
 from shared.versioning import generate_version
 
 TEMPLATE = """
--- a/tools/tr2/generate_init
+++ b/tools/tr2/generate_init
@@ -1,7 +1,12 @@
 #!/usr/bin/env python3
 import argparse
+import sys
+import os
 from pathlib import Path
 
+# Ajoute le répertoire parent au PYTHONPATH
+sys.path.insert(0, str(Path(__file__).resolve().parent.parent))
+
 from shared.versioning import generate_version
 
 TEMPLATE = """
